public/app/plugins/panel/geomap/style/markers.ts

```diff
@@ -115,13 +106,13 @@ const makers: SymbolMaker[] = [
   {
     id: RegularShapeId.circle,
     name: 'Circle',
-    aliasIds: [MarkerShapePath.circle],
+    aliasIds: ['img/icons/simple/circle.svg', 'img/icons/marker/circle.svg'],
```

Comment:
This dedicates the `img/icons/simple` folder for icons we can easily draw directly with canvas rather than the complex items that require loading the svg